crates/typst/src/geom/gradient.rs

```diff
@@ -31,6 +31,23 @@ use crate::syntax::{Span, Spanned};
 /// )
 /// ```
 ///
+/// # Gradient on text
+/// Gradients are supported on text but only when setting the relativeness to
+/// either `{auto}` or `{"parent"}`. Glyph-by-glyph text gradients will be
+/// supported in a future release. See the
+/// [tracking issue](https://github.com/typst/typst/issues/2282) for a more
+/// detailed status.
+///
+/// You can use gradients on text as follows:
+///
+/// ```example
+/// #set page(margin: 1pt)
+/// #set text(fill: gradient.linear(red, blue))
+/// #show box: set text(fill: gradient.linear(..color.map.rainbow))
```

Comment:
I think this `show box` isn't very idiomatic. Better to define a new function here.

Reply:
I changed it to a function but it has to be boxed to give the desired effect.